panel/src/components/Forms/Field/StructureField.vue

```diff
@@ -415,6 +412,8 @@ export default {
 				return false;
 			}
 
+			this.stopSelecting();
```

Comment:
`Explanation:` Would make sense to me not having the selecting UI being active while overlayed by the drawer.

Reply:
I think we should add this as a fix to develop-minor even develop-patch